input/resources/au-core-server.xml

```diff
@@ -4488,8 +4924,7 @@
 <definition value="http://hl7.org/fhir/SearchParameter/Practitioner-name"/>
 <type value="string"/>
 </searchParam>
-</resource>
-<!-- PRACTITIONERROLE :: DONE -->
+</resource><!-- PRACTITIONERROLE :: DONE -->
```

Comment:
Missing interactions for PractitionerRole

Reply:
Added